src/modules/common/components/input-dropdown/input-dropdown.jsx

```diff
@@ -61,7 +61,7 @@ class InputDropdown extends Component {
     return (
       <div
         ref={(InputDropdown) => { this.refInputDropdown = InputDropdown }}
-        className={Styles.InputDropdown}
+        className={classNames(Styles.InputDropdown, (p.className || ''))}
```

Comment:
No need for the conditional here — `classNames` will automatically filter out falsy values.